config/settings/local.py

```diff
@@ -13,11 +13,8 @@
     default="aHdCBMHXuxIxEhfRGFRp7Cp3N9CqEZEEAvwZVlBCazKExkEnzvVs4bYWC8Qqh9lg",
 )
 # https://docs.djangoproject.com/en/dev/ref/settings/#allowed-hosts
-ALLOWED_HOSTS = [
-    "localhost",
-    "0.0.0.0",
-    "127.0.0.1",
-]
+allowed_hosts = env("DJANGO_ALLOWED_HOSTS", default=".localhost,127.0.0.1,[::1]")
```

Comment:
Please use:

`ALLOWED_HOSTS = env("DJANGO_ALLOWED_HOSTS", default=[".localhost", "127.0.0.1", "[::1]"])`

No need for the next line

Reply:
Take a look at `production.py` 😉